tests/common.go

```diff
@@ -5221,3 +5221,34 @@ func GetRandomStorageLessNode(slNodes []node.Node) node.Node {
 	}
 	return node.Node{}
 }
+
+// Function will list all the pools associated with the volume
```

Comment:
Usually you follow format for comments with the function name first like so
`// GetPoolIDsFromVolName returns list of pool IDs associated with a given volume name`
NOTE: Just summarize what function takes and what function returns here, that way you don't need another comment describing it any further, then remove that comment at the beginning of this functions

Reply:
Thanks , updated the function comments as suggested